test/errors.cpp

```diff
@@ -148,6 +146,15 @@ void make_move_different_rank() { auto a2 = make_move(a, shape_of_rank<2>()); }
 
 void make_move_ref_different_rank() { auto a2 = make_move(ref, shape_of_rank<3>()); }
 
+// When commented out, this fails to compile as expected, but since the error
```

Comment:
Uncommented?

We *could* add an appropriate `enable_if` to `reinterpret_const` to make this test pass. I might try to do that at some point, it’s not necessary for now.

Reply:
Will do when I get home to a computer that has my ssh keys!

Reply:
Done.